### ISSUE TITLE
Not able to run on real pepper

### ISSUE THREAD

**Issue by sameer018** [aldebaran/qisdk-tutorials#3]
I'm not able to run on real pepper, i'm able to connect to pepper through ip. tablet is not detect on android studio.

**Comment by Siddounet** [comment]
Hi,
Got same issue, sometimes `adb kill-server` then `adb start-server` works, sometimes AS needs to be restarted.
Previously had this issue when emulator already connected (or connecting) while trying to use a real pepper.

**Comment by galfvens** [comment]
Also encountered such a problem, the simulator is normal, can connect to the pepper, but did not detect the device, when selecting the running target

**Comment by Siddounet** [comment]
Same here, only Android Studio reboot works when connected to pepper through adb but AS can't display nor select the emulator.

**Comment by pg12575** [comment]
Having same issue, any update on this? Thanks

**Comment by schuetz-dhbw** [comment]
unfortunetelly still an issue.. connection to pepper via android studio not possible (but worked before already)

**Comment by IarinaX** [comment]
Hello. I’m dealing with the same thing. Any updates ?

**Comment by schuetz-dhbw** [comment]
The integrated connection method doesn't work anymore for me. I'm using the plugin ADB Wi-Fi and connect manually to Pepper. This works pretty good.

**Comment by IarinaX** [comment]
Hello,

I just saw your answer on GIT. Do you happen to have any app on Pepper in
Android Studio? I am trying to make my diploma research on Pepper but I’m
having a really hard time finding anything useful.

Thank you!
Iarina

În joi, 4 mai 2023 la 13:14 schuetz-dhbw ***@***.***> a scris:

**Comment by IarinaX** [comment]
Something more,

Can you please tell me how you added the ADB Wi-Fi plugin?

Thank you
Iarina
În joi, 4 mai 2023 la 13:18 Iarina Măjeri ***@***.***> a
scris:

**Comment by schuetz-dhbw** [comment]
Yes, I created some apps for Pepper with Android Studio. It's not that easy as the documentation is bad and there's no support at all. Additonally, the OS is very old so it's not fun working with Pepper. But for small projects with basic functionality it's ok for me.
You have to add the ADB Wi-Fi plugin in Android Studio via Settings > Plugins. Then you have to enter Peppers IPs. 
I'm no pepper expert at all and it's just a side project for me sometimes but if you want to contact me directly via mail just have a look into my github profile.
